src/Cake.Core.Tests/Unit/Scripting/Processors/LoadDirectiveProcessorTests.cs

```diff
@@ -137,5 +138,43 @@ public void Should_Insert_Line_Directives_When_Processing_Load_Directives()
             Assert.Equal(result.Lines[11], "// #l b.cake");
             Assert.Equal(result.Lines[12], "int y=2;");
         }
+
+        [Theory]
+        [InlineData("#load \"/utils.cake\"")]
+        [InlineData("#load \"local:?path=/utils.cake\"")]
+        public void Should_Process_AbsolutePath_Script_Reference_Found_In_Source(string source)
+        {
+            // Given
+            var fixture = new ScriptAnalyzerFixture();
+            fixture.Providers.Add(new FileLoadDirectiveProvider());
+            fixture.GivenScriptExist("/Working/script.cake", source);
+            fixture.GivenScriptExist("/utils.cake", "Console.WriteLine();");
+
+            // When
+            var result = fixture.Analyze("/Working/script.cake");
+
+            // Then
+            Assert.Equal(1, result.Script.Includes.Count);
+            Assert.Equal("/utils.cake", result.Script.Includes[0].Path.FullPath);
+        }
+
+        [WindowsTheory]
+        [InlineData("#load \"c:/utils.cake\"")]
+        [InlineData("#load \"local:?path=c:/utils.cake\"")]
+        public void Should_Process_WindowsAbsolutePath_Script_Reference_Found_In_Source(string source)
```

Comment:
Can we also add tests for relative file paths?

Reply:
Done